lib/imap.php

```diff
@@ -72,6 +72,10 @@ public function checkPassword($uid, $password) {
 					$uid = $pieces[0];
 				}
 			} else {
+				OC::$server->getLogger()->error(
+				'ERROR: Wrong domain: '.$this->domain,
+				['app' => 'user_external']
+			);
```

Comment:
could you indent the content of `error()` function as well as the closing parenthesis one tab more? like you have done in the changes below...

Reply:
thanks for changing!